presto-parser/src/main/java/com/facebook/presto/sql/tree/SimpleGroupBy.java

```diff
@@ -42,7 +42,7 @@ public SimpleGroupBy(NodeLocation location, List<Expression> simpleGroupByExpres
     private SimpleGroupBy(Optional<NodeLocation> location, List<Expression> simpleGroupByExpressions)
     {
         super(location);
-        this.columns = requireNonNull(simpleGroupByExpressions);
+        this.columns = ImmutableList.copyOf(requireNonNull(simpleGroupByExpressions, "simpleGroupByExpressions is null"));
```

Comment:
Doing it inline is better when you have multiple items. Otherwise, it's hard to tell that you didn't miss one. Or you interleave them, which is also hard to read:

```java
this.foo = ...
requireNonNull(sessionProperties, "sessionProperties is null");
this.sessionProperties = ImmutableList.copyOf(sessionProperties);
requireNonNull(tableProperties, "tableProperties is null")
this.tableProperties = ImmutableList.copyOf(tableProperties);
```

So maybe add newlines:
```java
this.foo = ...

requireNonNull(sessionProperties, "sessionProperties is null");
this.sessionProperties = ImmutableList.copyOf(sessionProperties);

requireNonNull(tableProperties, "tableProperties is null")
this.tableProperties = ImmutableList.copyOf(tableProperties);
```

And now you've gone from 2 lines to 6, for something that had arguably no benefit in the first place: for `ImmutableList.copyOf()`, the check isn't needed at all, since `copyOf()` will throw on null and the caller will have a stack trace pointing to the exact line. The only benefit is a better error message.